### PR TITLE
[WIP] Manual override for IPv6 support

### PR DESCRIPTION
**Pull request by caseydavenport**
Marking as high priority since this affects our ability to run in the default kube-up GCE provider (which doesn't support IPv6)

### CODE REVIEW

**Comment by caseydavenport** [comment]
I've tested that this fixes my direct issue (kube-up GCE provider).  @fasaxc @matthewdupre could you take a look?

File: calico/felix/futils.py [hunk 1]
```diff
@@ -541,6 +541,13 @@ def ipv6_supported():
         return False, ("ip6tables not installed; Calico IPv6 support requires "
                        "Linux kernel v3.3 or above and ip6tables v1.4.14 or "
                        "above.")
+
+    # Check for the existence of the IPv6 NAT table.
+    try:
+        check_call(["ip6tables-save", "--table", "nat"])
```
Comment (fasaxc) [inline calico/felix/futils.py]: Please add a UT for this case.  Should be a matter of `m_check_call.side_effect = iter([None, FailedSystemCall(...)])`

**Comment by fasaxc** [comment]
retest this please

File: calico/felix/config.py [hunk 2]
```diff
@@ -773,6 +782,12 @@ def _validate_cfg(self, final=True):
                     raise ConfigException("Out-of-range port %s" % p,
                                           self.parameters[name])
 
+        if self.IPV6_SUPPORT not in ("true", "false", "auto"):
+            raise ConfigException(
```
Comment (fasaxc) [inline calico/felix/config.py]: You should also add a UT for this

**Comment by lwr20** [comment]
retest this please

**Comment by lwr20** [comment]
retest this please

**Comment by lwr20** [comment]
retest this please

**Comment by lwr20** [comment]
retest this please

**Comment by lwr20** [comment]
retest this please

**Comment by lwr20** [comment]
retest this please

**Comment by caseydavenport** [comment]
OK  - Latest commit should address comments.

**Comment by fasaxc** [comment]
LGTM

**Pull request merged**